# Add this patch to fix error in newer macOS versions
--- src/hotspot/share/gc/parallel/psParallelCompact.cpp.orig   2022-03-12 20:43:17.000000000 +0530
+++ src/hotspot/share/gc/parallel/psParallelCompact.cpp        2022-03-12 20:46:59.000000000 +0530
@@ -1611,16 +1611,17 @@
 {
   GCTraceTime(Info, gc, phases) tm("Summary Phase", &_gc_timer);
 
-  log_develop_debug(gc, marking)(
-      "add_obj_count=" SIZE_FORMAT " "
-      "add_obj_bytes=" SIZE_FORMAT,
-      add_obj_count,
-      add_obj_size * HeapWordSize);
-  log_develop_debug(gc, marking)(
-      "mark_bitmap_count=" SIZE_FORMAT " "
-      "mark_bitmap_bytes=" SIZE_FORMAT,
-      mark_bitmap_count,
-      mark_bitmap_size * HeapWordSize);
+#ifdef  ASSERT
+  if (TraceParallelOldGCMarkingPhase) {
+    tty->print_cr("add_obj_count=" SIZE_FORMAT " "
+                  "add_obj_bytes=" SIZE_FORMAT,
+                  add_obj_count, add_obj_size * HeapWordSize);
+    tty->print_cr("mark_bitmap_count=" SIZE_FORMAT " "
+                  "mark_bitmap_bytes=" SIZE_FORMAT,
+                  mark_bitmap_count, mark_bitmap_size * HeapWordSize);
+  }
+#endif  // #ifdef ASSERT
+
 
   // Quick summarization of each space into itself, to see how much is live.
   summarize_spaces_quick();
